### ISSUE TITLE
Balsa: needs openldap2.6

### ISSUE THREAD

**Issue by humdingerb** [haikuports/haikuports#9262]
When launching Balsa, it complained:
```
❯ /boot/system/apps/Balsa
runtime_loader: Cannot open file libldap.so.2 (needed by /boot/system/apps/Balsa): No such file or directory
```

I had `openldap` installed, but that apparently didn't satisfy it.
After installing `openldap2.6`, it worked.

**Comment by Begasus** [comment]
I've got a PR open for this (openldap: https://github.com/haikuports/haikuports/pull/9260), this should address the issue (problem is that openldap2.4* PROVIDES lib:libldap in the recipe (but it only provides lib:libldap_r), openldap2.6.* only provides lib:libldap (well both have the equivelant liblber version too), hence the issue here.
openldap2.4.* is needed for LO, so we need to have a package providing the libraries for it so it doesn't break LO.
gnupg depends on lib:ldap and currently is linked against openldap2.4.*, so that needs to be fixed, for that I got an update ready for a newer version (that needs newer versions for 3 other libraries), so untill this is fixed one can install openldap2.6 to bypass thee issue as you mentioned (already mentioned at the forum too). :)
